### DISCUSSION TITLE
correct date for last update not correct on meeting list

### DISCUSSION THREAD

**Discussion by CheriJones** [code4recovery/12-step-meeting-list#1550] [General]
when we update a meeting listing the date in the "Last Contact" field of the meeting edit page is correct - but when we publish the changes to the meetings doesn't show the new publish date - example - wamo-aa.org - Thursday in liberty - liberty online group info updated 10/24/2024 - meeting page shows Updated May 18, 2023

**Comment by joshreisner** [comment]
the "last contact" field is something else - that's an optional date field for sites to record their last contact with a group

the updated date is when the meeting was last updated - wordpress keeps track of this automatically. if you click the blue Update button it should say today's date. there's no way i'm aware of to actually see this date on the edit screen though.

i'm guessing you some kind of caching plugin installed? i see it's still still showing the old date on the [meeting detail page
](https://wamo-aa.org/meetings/6866/?tsml-day=4&tsml-query=in%20liberty) but i see it showing today's date in the JSON feed:
<img width="567" alt="Screenshot 2024-10-24 at 1 09 32 PM" src="https://github.com/user-attachments/assets/2ba0fdf5-6b3d-457f-ad2b-b65f160e85f9">

**Reply by CheriJones** [reply]
i didn't think we had a cashing plugin and i don't see anything obvious in the plugin list - what should i be looking for

**Reply by joshreisner** [reply]
looks like you've got this plugin installed: https://wordpress.org/plugins/sg-cachepress/

**Reply by CheriJones** [reply]
i manually cleared the cache thru the plugin and reupdated the liberty Thursday meeting - no change
then i added the meeting URL to excluded from cache and reupdated the meeting - no change
maybe i'm doing it wrong - i'm really shooting in the dark - suggestions ??

**Reply by joshreisner** [reply]
can you try deactivating that plugin?

**Comment by CheriJones** [comment]
found it finally - the host installed it i guess automatically - i'll start looking into it - thanks for the help

**Comment by CheriJones** [comment]
i manually cleared the cashe and updated a meeting - shooting in the datk here - didn't make a change to the meeting detail page - suggestions ??

**Reply by joshreisner** [reply]
i don't think that worked. maybe deactivate the plugin?

**Reply by gkovats** [reply]
@CheriJones still having issues? Or this is working now?
